integration_tests/mockApis/responses/allPrisons.ts

```diff
@@ -1,6 +1,4 @@
-import { Prison } from '../@types/bapv'
-
-const prisons: Prison[] = [
+export default [
```

Comment:
Git seems to be suggesting I've moved this file - not quite what's happened 🙄! Have made this `responses` folder here to start trying to separate the test data from the stub API definitions.